### PR TITLE
docs: Update third-party-libraries.md

### PR DESCRIPTION
**Pull request by AwesomeDevin**
Update third-party-libraries：
- zustand-vue - State management for vue (Vue3 / Vue2) based on zustand. 
- zustand-pub - Cross-Application/Cross-Framework State Management And Sharing based on zustand and zustand-vue for React/Vue.

### CODE REVIEW

**Review by dai-shi — CHANGES_REQUESTED** [review]
Thanks. Can you put them in the alphabetical order?

**Comment by AwesomeDevin** [comment]
> alphabetical order

OK

**Comment by AwesomeDevin** [comment]
> Thanks. Can you put them in the alphabetical order?

it’s done~

File: docs/integrations/third-party-libraries.md [hunk 1]
```diff
@@ -37,10 +37,13 @@ This can be done using third-party libraries created by the community.
...
+- [zustand-pub](https://github.com/AwesomeDevin/zustand-pub) - Cross-Application/Cross-Framework State Management And Sharing based on zustand and zustand-vue for React/Vue.   
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
 - [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
+- [zustand-vue](https://github.com/AwesomeDevin/zustand-vue) - State management for vue (Vue3 / Vue2) based on zustand.   
 - [zustand-yjs](https://github.com/tandem-pt/zustand-yjs) — Zustand stores for Yjs structures.
 - [zusteller](https://github.com/timkindberg/zusteller) — Your global state savior. "Just hooks" + Zustand.
 - [zustood](https://github.com/udecode/zustood) — 🐻‍❄️ A modular store factory using Zustand.
 - [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
 - [zustand-querystring](https://github.com/nitedani/zustand-querystring) — A Zustand middleware that syncs the store with the querystring.
```
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:48]: Ah, it wasn't sorted previously either.

```suggestion
- [zustand-querystring](https://github.com/nitedani/zustand-querystring) — A Zustand middleware that syncs the store with the querystring.
- [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
- [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
- [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.
- [zustand-vue](https://github.com/AwesomeDevin/zustand-vue) - State management for vue (Vue3 / Vue2) based on zustand.   
- [zustand-yjs](https://github.com/tandem-pt/zustand-yjs) — Zustand stores for Yjs structures.
- [zusteller](https://github.com/timkindberg/zusteller) — Your global state savior. "Just hooks" + Zustand.
- [zustood](https://github.com/udecode/zustood) — 🐻‍❄️ A modular store factory using Zustand.
```

File: docs/integrations/third-party-libraries.md [hunk 2]
```diff
@@ -37,10 +37,13 @@ This can be done using third-party libraries created by the community.
 - [zustand-middleware-xstate](https://github.com/biowaffeln/zustand-middleware-xstate) — A middleware for putting XState state machines into a global Zustand store.
 - [zustand-middleware-yjs](https://github.com/joebobmiles/zustand-middleware-yjs) — A middleware for synchronizing Zustand stores with Yjs.
 - [zustand-persist](https://github.com/roadmanfong/zustand-persist) — A middleware for persisting and rehydrating state.
+- [zustand-pub](https://github.com/AwesomeDevin/zustand-pub) - Cross-Application/Cross-Framework State Management And Sharing based on zustand and zustand-vue for React/Vue.   
 - [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
```
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:41]: Ah, it wasn't sorted previously anyway.
```suggestion
- [zustand-querystring](https://github.com/nitedani/zustand-querystring) — A Zustand middleware that syncs the store with the querystring.
- [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
- [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
```

File: docs/integrations/third-party-libraries.md (hunk 1 above)
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:48]:
```suggestion
```

**Comment by dai-shi** [comment]
Can you adjust them too?

File: docs/integrations/third-party-libraries.md (hunk 2 above)
Comment (AwesomeDevin) [reply docs/integrations/third-party-libraries.md:41]: > Can you adjust them too?

sure.

**Comment by AwesomeDevin** [comment]
> Can you adjust them too?

Please review~

**Review by dai-shi — APPROVED** [review]
Thanks for your help!

**Comment by AwesomeDevin** [comment]
Thank you. I submitted your code

**Comment by dai-shi** [comment]
Hmmm, not sure what CI is complaining. Can you run `yarn run prettier`?

**Comment by AwesomeDevin** [comment]
> Hmmm, not sure what CI is complaining. Can you run `yarn run prettier`?

👌

**Pull request closed without merging**